### PR TITLE
Share state in FP: fix typo

### PR DESCRIPTION
**Pull request by guizmaii**

### CODE REVIEW

**Comment by larsrh** [comment]
Thanks!

File: posts/2018-06-07-shared-state-in-fp.md [hunk 1]
```diff
@@ -267,7 +267,7 @@ class IORef {
 }
 ```
 
-This is way better since now the mutation is encapsulated within `IORef` but we are now pushing some resposibility to whoever creates an `IORef`. Consider this:
+This is way better since now the mutation is encapsulated within `IORef` but we are now pushing some responsibility to whoever creates an `IORef`. Consider this:
```
Comment (guizmaii) [inline posts/2018-06-07-shared-state-in-fp.md]: I'm not a native english speaker but for me, `responsibility` should be pluralize here, no ? 

> **some** responsibility

The "some" here indicates that there're more than one responsibility, no ?

Comment (larsrh) [reply posts/2018-06-07-shared-state-in-fp.md]: There's the phrase "to take some responsibility", so plural is not necessary here.

Comment (guizmaii) [reply posts/2018-06-07-shared-state-in-fp.md]: Ok thanks Lars

**Pull request merged**